src/scss/blocks/_box.scss

```diff
@@ -3,7 +3,7 @@
 .box {
   border: 1px solid get-color('secondary-shade');
   box-shadow: 0 0 3px 0 rgb(0 0 0 / 20%);
-  padding: get-size('600') get-size('400');
+  padding: get-size('500');
```

Comment:
This change will affect every use of this class so the effects will extend beyond the intended usage here. You could add `box--contact-us` or if you want something more generic, maybe `box--equal-space`. That would allow you to add a `box--equal-space` class to override the padding as you are doing here without affecting the existing uses. Maybe there is a better name, I am open to that. Just trying to make this change in a safe way.